### PR TITLE
[MW-4] Adding team component with everyone's information

### PR DESCRIPTION
**Pull request by LenaMubarak**
* **Please check if the PR fulfills these requirements**
- [ ] The pull request title contains the ticket number in the title. For example [CB-1]: MW-4-team

* **What kind of change does this PR introduce?** New feature: Team Components, cards with each member's information

* **What is the current behavior?What does the code do before you made the new changes?** No display

* **What is the new behavior?** Display 5 team card components

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR? Run a new command?) None. 

* **Other information**:

### CODE REVIEW

File: frontend/src/views/LandingPage.js [hunk 1]
```diff
@@ -1,10 +1,14 @@
 import React from 'react';
+import Team from "../components/Team";
 
 function LandingPage(){
     return(
         <div>
-            {/* Insert your created components here */}
-            <h1>Landing Page Component</h1>
+            {/* Insert your created components here */
+                //team component
```
Comment (Codubee) [inline frontend/src/views/LandingPage.js]: You can remove these comments as they add too much value to the reader.

File: frontend/src/views/LandingPage.js [hunk 2]
```diff
@@ -1,10 +1,14 @@
...
+                <Team/> 
+            }
+            <h1>Landing Page Component</h1>
```
Comment (Codubee) [inline frontend/src/views/LandingPage.js]: Remove this as well.

File: frontend/src/views/LandingPage.js [hunk 3]
```diff
@@ -1,10 +1,14 @@
...
+import Team from "../components/Team";
 
 function LandingPage(){
     return(
         <div>
-            {/* Insert your created components here */}
-            <h1>Landing Page Component</h1>
+            {/* Insert your created components here */
+                //team component
+                <Team/>
```
Comment (Codubee) [inline frontend/src/views/LandingPage.js]: The component does not have to be inside of curly braces.

File: frontend/src/components/Team.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+import{
+    Card, CardImg, CardText, CardBody, CardTitle, CardSubtitle, CardDeck
+} from 'reactstrap'
+
+//components
+function team() {
+    return(
+        <div>
+            <CardDeck>
+                <Card style={{width:"350px", height:"650px"}} color='success'>
```
Comment (Codubee) [inline frontend/src/components/Team.js]: This comment doesnt help the reader much. Consider removing.

File: frontend/src/components/Team.js [hunk 5]
```diff
@@ -0,0 +1,74 @@
+//Creates the team component 
+
+//import react
```
Comment (Codubee) [inline frontend/src/components/Team.js]: This comment doesnt help the reader much. Consider removing.

File: frontend/src/components/Team.js [hunk 6]
```diff
@@ -0,0 +1,74 @@
+//Creates the team component
```
Comment (Codubee) [inline frontend/src/components/Team.js]: This comment doesnt help the reader much. Consider removing.

File: frontend/src/components/Team.js [hunk 7]
```diff
@@ -0,0 +1,74 @@
+//Creates the team component 
+
+//import react
+import React from 'react'
+
+//import images
```
Comment (Codubee) [inline frontend/src/components/Team.js]: This comment doesnt help the reader much. Consider removing.

File: frontend/src/components/Team.js [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+import lena from '../images/lena.jpg'
+import alex from '../images/alex.png'
+import arielle from '../images/arielle.jpg'
+import hannah from '../images/hannah.jpg'
+import arath from '../images/arath.jpg'
+
+
+//import a card component
```
Comment (Codubee) [inline frontend/src/components/Team.js]: This comment doesnt help the reader much. Consider removing.

File: frontend/src/components/Team.js [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+import{
+    Card, CardImg, CardText, CardBody, CardTitle, CardSubtitle, CardDeck
+} from 'reactstrap'
+
+//components
```
Comment (Codubee) [inline frontend/src/components/Team.js]: This comment doesnt help the reader much. Consider removing.

File: frontend/src/components/Team.js [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+                        <CardSubtitle tag="h6">Mechanical Engineering</CardSubtitle> 
+                        <CardText>The University of Texas at Dallas <p>Alumnus</p> </CardText>
+                    </CardBody>
+                </Card>
+            </CardDeck>
+        </div>
+    )
+}
+
+//export component to use in app
```
Comment (Codubee) [inline frontend/src/components/Team.js]: This comment doesnt help the reader much. Consider removing.

**Review by HanaEason — APPROVED** [review]
Everything looks good to me!

**Pull request merged**